tutor/src/theme.js

```diff
@@ -3,7 +3,7 @@ import { css } from 'styled-components';
 // Shared with scss in styles/variables
 
 const neutral = {
-  lightest: '#f9f9f9', // nearly white
+  lightest: '#fafafa', // nearly white
```

Comment:
`fafafa` is used a lot in new designs and `lightest` is only 1 off and previously unused, so changing it permanently here.